clang/lib/Driver/ToolChains/HIPUtility.cpp

```diff
@@ -466,7 +467,7 @@ void HIP::constructGenerateObjFileFromHIPFatBinary(
                        "-o",      Output.getFilename(),
                        "-x",      "assembler",
                        ObjinFile, "-c"};
-  const char *Clang = Args.MakeArgString(C.getDriver().ClangExecutable);
   C.addCommand(std::make_unique<Command>(JA, T, ResponseFileSupport::None(),
-                                         Clang, McArgs, Inputs, Output));
+                                         D.getClangProgramPath(), McArgs,
```

Comment:
Guess we don't need to `MakeArgString` because the lifetime of the Driver is for the whole compilation?

Reply:
Yes, this is what we do in other parts of the driver as well.